src/index.html

```diff
@@ -1,17 +1,108 @@
     >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" >
     <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./style.css" >
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="groups">
+        <legend>Personal information</legend>
+        <label class="form-field"> Surname:
+          <input
+          type="text"
+          name="Surname:"
+          autocomplete="off"
+          >
+        </label>
+        <label class="form-field"> Name:
+          <input
+          type="text"
+          name="Name:"
+          autocomplete="off"
+          >
+        </label>
+        <label class="form-field"> How old are You?
+          <input type="number"
+          name="age"
+          class="age"
+          min="1"
+          max="100"
+          value="12"
+          >
+        </label>
+        <label class="form-field"> Full date of birth: <input type="date" name="birthday"> </label>
+        <label class="form-field">
+          I accept the term of the agreement
+          <input
+          type="checkbox"
+          name="IAcceptTheTermOfThrAgreement"
+          required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="groups">
+        <legend>Registration</legend>
+        <label class="form-field">
+          E-mail: <input type="email" placeholder="email@example.com" >
+        </label>
+        <label class="form-field"> Password:
+          <input
+          type="password"
+          minlength="6"
+          maxlength="10"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="groups">
+        <legend>An interesting fact about you!</legend>
+        <label class="form-field">
+          Do you love cats?
+          <label>
+            <input type="radio" name="Do you love cats?"> Yes
+          </label>
+          <label>
+            <input type="radio" name="Do you love cats?"> No
+          </label>
+        </label>
+        <label class="form-field"> What is your favorite color? <input type="color" name="What is your favorite color?"> </label>
+        <label class="form-field"> What time do you go to bed? <input type="time" name="What time do you go to bed?"> </label>
+        <label class="form-field">
+          What are your favorite brend of car?
+          <select name="WhatAreYourFavoriteBrendOfCar?" multiple>
+            <option value="BMW">BMW</option>
+            <option value="Audi" selected>Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+        <label class="form-field"> How do you rate our work? <input type="range"  name="How do you rate our work?"> </label>
```

Comment:
```suggestion
        <label class="form-field">How do you rate our work?
          <input type="range" name="How do you rate our work?">
        </label>
```
too long for one line, max length is 80. fix in other places